dashboard/app/views/api/user_hero.html.haml

```diff
@@ -19,7 +19,7 @@
             .flex_item{style: 'width: 50%'}
               #{course_enrollment.plc_course.name}
             .flex_item{style: 'width: 50%'}
-              = link_to course_path(course_enrollment.plc_course.name.downcase.sub(' ', '-')) do
+              = link_to root_url + course_path(course_enrollment.plc_course.name.downcase.gsub(' ', '-')) do
```

Comment:
Does this work in all environments (localhost/staging/test/prod) to keep the user on that environment?  We have some other helpers that do that at https://github.com/code-dot-org/code-dot-org/blob/staging/deployment.rb#L200

But if it already works, then lgtm.